sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DeserializationHelpers.java

```diff
@@ -1,23 +1,37 @@
 package com.azure.digitaltwins.core.implementation.serializer;
 
+import com.azure.core.util.serializer.JsonSerializer;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+
+import static com.azure.core.util.serializer.TypeReference.createInstance;
+
 public class DeserializationHelpers {
 
     /**
-     * Converts the payload object into a generic type.
+     * Deserialize the payload object into a generic type.
      * There are two different paths we will have to take based on the type T
      * In case of a String, we need to write the value of the payload as a String
      * In case of any other type that the user decides to deserialize the payload, we will use mapper.convertValue to perform the conversion.
+     *
+     * If the customJsonSerializer is null, then no custom serializer will be used for this deserialization. Otherwise, the customJsonSerializer will be used.
```

Comment:
shouldn't we say what would be used instead, if customJsonSerializer is null?

Reply:
Let me re-word this a bit to avoid confusion